index.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
```

Comment:
- [ ] optional.. maybe the previous reviewer forgot to mention it.. from the project requirement the menu popup needs a close button or icon. please lets endeavours to make it available as it will ease accesibility of the portfolio.
 your menu 👇🏾 ;
![a (2)_LI](https://user-images.githubusercontent.com/65298048/169578655-3d459ca0-5ed2-4259-a0ec-1727990526d6.jpg)
-------
`Figma` design menu ;

![b (2)_LI](https://user-images.githubusercontent.com/65298048/169578756-7ec45de4-4c2e-4659-9216-08f170494b85.jpg)